app/tests/spec/views/base.js

```diff
@@ -219,7 +215,7 @@ define(function (require, exports, module) {
           sessionToken: 'abc123',
           uid: 'uid'
         });
-        sinon.stub(fxaClient, 'sessionStatus', function () {
+        sinon.stub(account, 'isSignedIn', function () {
```

Comment:
Good catch, probably should. In my laziness I only updated the stub and didn't add a test.